### PR TITLE
feat(route): add a new router for radio france

### PR DESCRIPTION
**Pull request by xdu**
## 该 PR 相关 Issue / Involved Issue

Close #

## 路由地址示例 / Example for the Proposed Route(s)

```routes
/radiofrance/geopolitique
```

## 新 RSS 路由检查表 / New RSS Route Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [x] 全文获取 fulltext
  - [x] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: docs/traditional-media.md [hunk 1]
```diff
@@ -2706,3 +2706,13 @@ category 对应的关键词有
 | 1           | 2           | 3           | 4               | 5               | 6               | 7               | 8           |
 
 </Route>
+
+## 法国广播电台
+
+### 地缘政治栏目
+
+<Route author="xdu" example="/radiofrance/geopolitique" path="/radiofrance/geopolitique" supportPodcast="0">
```
Comment (TonyRL) [inline docs/traditional-media.md:2714]:
```suggestion
<Route author="xdu" example="/radiofrance/geopolitique" path="/radiofrance/geopolitique">
```
Don't add the attribute if the does not support it.

File: lib/v2/radiofrance/geopolitique.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+					const apiurl = 'https://www.radiofrance.fr/api/v2.1/path?value=' + encodeURIComponent(path);
+					const { data } = await got(apiurl);
+
+					delete item.content;
+					delete item.contentSnippet;
+
+					item.description = '<p>' + data.content.standFirst + '</p>'
+						+ data.content.bodyJson.map((t) =>
+							'<p>' + t.children[0].value + '</p>'
+						).join('');
```
Comment (TonyRL) [inline lib/v2/radiofrance/geopolitique.js:22]: Please use `art-template` for custom HTML.
docs: https://docs.rsshub.app/joinus/script-standard.html

File: docs/traditional-media.md [hunk 3]
```diff
@@ -2706,3 +2706,13 @@ category 对应的关键词有
...
+
```
Comment (TonyRL) [inline docs/traditional-media.md:2715]: Missing en docs.

Comment (xdu) [reply docs/traditional-media.md:2715]: Hi, what is missing en docs ? 😃

Comment (TonyRL) [reply docs/traditional-media.md:2715]: Missing English docs.

Comment (xdu) [reply docs/traditional-media.md:2715]: okay thanks. now added.

File: docs/en/traditional-media.md [hunk 4]
```diff
@@ -355,6 +355,16 @@ Only `s00017` is in English.
 
 </RouteEn>
 
+## Radio France
+
+### Géopolitique
+
+<Route author="xdu" example="/radiofrance/geopolitique" path="/radiofrance/geopolitique">
```
Comment (TonyRL) [inline docs/en/traditional-media.md:362]:
```suggestion
<RouteEn author="xdu" example="/radiofrance/geopolitique" path="/radiofrance/geopolitique">
```

File: docs/en/traditional-media.md [hunk 5]
```diff
@@ -355,6 +355,16 @@ Only `s00017` is in English.
...
+
+French podcast on the international politics. This feed provides a better reading experience (full text) for the 3 latest articles.
+
+</Route>
```
Comment (TonyRL) [inline docs/en/traditional-media.md:366]:
```suggestion
</RouteEn>
```

File: lib/v2/radiofrance/geopolitique.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+				})));
+
+	ctx.state.data = {
+		title: feed.title,
+		link: feed.link,
+		description: feed.description,
+		item: items,
+		language: feed.language,
+	};
+}
```
[the remark this answers is not in the document]
Comment (xdu) [reply lib/v2/radiofrance/geopolitique.js]: Sorry for those lint issues, now they are fixed. I hope it will be fine this time. Thanks

**Pull request merged**